addon/plugins/article-structure-plugin/commands/insert-structure.ts

```diff
@@ -80,21 +85,26 @@ function findInsertionRange(args: {
       }
     }
   }
-  const nextContainerRange = findNodes(
-    doc,
-    selection.from,
-    true,
-    false,
-    ({ from }) => {
+  const limitContainer = limitTo
+    ? findParentNodeOfType(schema.nodes['limitTo'])(selection)
```

Comment:
I think this limitTo has to be without quotes right?

Reply:
you are right!